### PR TITLE
Cjl swagger endpoint

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

**Comment by geoffjentry** [comment]
:+1: post-squash

File: src/main/scala/thurloe/service/ThurloeServiceActor.scala [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package thurloe.service
 
 import akka.actor.{ActorRefFactory, ActorLogging}
+import slick.collection.heterogeneous.HList
+import spray.routing
```
Comment (mcovarr) [inline src/main/scala/thurloe/service/ThurloeServiceActor.scala]: these look unused

File: src/main/scala/thurloe/service/ThurloeService.scala [hunk 2]
```diff
@@ -20,6 +20,10 @@ trait ThurloeService extends HttpService {
   val dataAccess: DataAccess
   val thurloePrefix = "thurloe"
 
+  // TODO: Relies on swagger existing in resources/swagger/
+  val swaggerSite = path("swagger-site") { getFromResource("swagger/index.html") } ~ getFromResourceDirectory("swagger")
+  val apiSpecYaml = path("swagger" / "thurloe.yaml") { getFromResource("yaml/thurloe-api.yaml") }
```
Comment (mcovarr) [inline src/main/scala/thurloe/service/ThurloeService.scala]: is there a reason to mismatch the name of the route and the actual file?

**Comment by geoffjentry** [comment]
Since conditions on the ground are actively changing let's hold off on this until we tackle DSDEEPB-1445

File: src/main/scala/thurloe/service/ThurloeService.scala [hunk 3]
```diff
@@ -20,6 +20,10 @@ trait ThurloeService extends HttpService {
   val dataAccess: DataAccess
   val thurloePrefix = "thurloe"
 
+  // TODO: Relies on swagger existing in resources/swagger/
+  val swaggerSite = path("swagger-site") { getFromResource("swagger/index.html") } ~ getFromResourceDirectory("swagger")
```
Comment (kshakir) [inline src/main/scala/thurloe/service/ThurloeService.scala]: In addition to @mcovarr's comment about moving yaml file to match the the endpoint, I don't believe for https://swagger.dsde-dev.broadinstitute.org we need this particular "swagger-site" endpoint, nor the `"org.webjars" % "swagger-ui" % "2.1.1"` in thurloe's build.sbt, only the thurloe.yaml file. In fact, a remaining cromwell TODO is to remove `swagger-ui` and `spray-swagger` now that the yaml appears to be working.

Comment (geoffjentry) [inline src/main/scala/thurloe/service/ThurloeService.scala]: We're supposed to have swagger locally, unrelated to swagger.dsde-dev

Comment (kshakir) [inline src/main/scala/thurloe/service/ThurloeService.scala]: Whoops, well the swagger-ui exposed endpoint has been removed from cromwell as of a few commits back. We should probably have a test here and in cromwell for checking that `/swagger/index.html` returns the expected content from the ui jar. For now… this line doesn't really do anything but return 404s for various reasons. :wink: [This](https://github.com/broadinstitute/cromwell/commit/f82eaa5cbcc7ed64f534c671c13a2f9cdb1df5f1#diff-e25accda21d47a99a8060d7e9f4d32a5) commit has a ui endpoint that used to serve up the ui jar content. (Note: It also included dynamic spray-swagger generating the swagger json, that I believe we are _definitely_ not using anymore.)

Comment (geoffjentry) [inline src/main/scala/thurloe/service/ThurloeService.scala]: Yeah, this whole thing is confusing and isn't helped by it being ill defined currently in the dept. There's a ticket now to get us conforming to the standard and we can address it when we get there.

**Pull request closed without merging**